below/render/src/default_configs.rs

```diff
@@ -143,6 +143,118 @@ impl HasRenderConfigForDump for model::SingleCgroupModel {
         }
         .get()
     }
+
+    fn get_openmetrics_config_for_dump(
+        &self,
+        field_id: &Self::FieldId,
+    ) -> Option<RenderOpenMetricsConfigBuilder> {
+        use model::CgroupCpuModelFieldId::*;
+        use model::CgroupIoModelFieldId::*;
+        use model::CgroupMemoryModelFieldId::*;
+        use model::CgroupPressureModelFieldId::*;
+        use model::CgroupStatModelFieldId::*;
+        use model::SingleCgroupModelFieldId::*;
+
+        let counter = counter().label("cgroup", &self.full_path);
+        let gauge = gauge().label("cgroup", &self.full_path);
+        match field_id {
+            // We only use full path for the label to avoid ambiguity
+            Name => None,
+            // We will label each metric with the full path
+            FullPath => None,
+            // Not sure what to do with static fields like inode number so leave out for now
+            InodeNumber => None,
+            Cpu(field_id) => match field_id {
+                UsagePct => Some(gauge.unit("percent")),
+                UserPct => Some(gauge.unit("percent")),
+                SystemPct => Some(gauge.unit("percent")),
+                NrPeriodsPerSec => Some(gauge),
+                NrThrottledPerSec => Some(gauge),
+                ThrottledPct => Some(gauge.unit("percent")),
+            },
+            Io(field_id) => match field_id {
+                RbytesPerSec => Some(gauge.unit("bytes")),
```

Comment:
These are really bytes per second, not just bytes, right?

Reply:
Ah yeah